src/aleph/vm/sevclient.py

```diff
@@ -4,20 +4,27 @@
 
 
 class SevClient:
-    def __init__(self, sev_dir: Path):
+    sev_dir: Path
+    sev_ctl_executable: Path
+    certificates_dir: Path
+    certificates_archive: Path
+
+    def __init__(self, sev_dir: Path, sev_ctl_executable: Path):
         self.sev_dir = sev_dir
+        self.sev_ctl_executable = sev_ctl_executable
```

Comment:
In that case, it would be better to set `settings.SEV_CTL_PATH` to the version of `sevctl` detected on the system in the setup of the settings.

Reply:
This branch uses the same behavior as for `FIRECRACKER_PATH`, `JAILER_PATH` and `LINUX_PATH`.

Changing this behavior is a topic different from this PR as it would have other consequences (what is the default, are fallback values provided, ...).

Reply:
Okay, so we can continue at the moment and improve it later.